fedn/fedn/clients/reducer/control.py

```diff
@@ -74,12 +74,18 @@ def get_latest_model(self):
     def get_model_info(self):
         return self.statestore.get_model_info()
 
+    def get_client_info(self):
```

Comment:
I think this should belong in network.py

Reply:
I see, done!